src/modules/charging-import/plugin.js

```diff
@@ -2,15 +2,19 @@ const cron = require('node-cron');
 const jobs = require('./jobs');
 
 const chargingImport = require('./lib/import');
+const chargeVersionMetadataImport = require('./lib/import-charge-version-metadata');
+
 const { createRegister } = require('../../lib/plugin');
 const config = require('../../../config');
 
 const registerSubscribers = async server => {
-  // Import charging data
+  // Register PG boss job handlers
   await server.messageQueue.subscribe(jobs.IMPORT_CHARGING_DATA, chargingImport.importChargingData);
+  await server.messageQueue.subscribe(jobs.IMPORT_CHARGE_VERSION_METADATA, chargeVersionMetadataImport.importChargeVersionMetadata);
```

Comment:
I'm wondering if `IMPORT_CHARGE_VERSION_METADATA` should happen before `IMPORT_CHARGING_DATA` rather than after.

Reply:
This code is the subscribers being bound rather than the order of jobs.
However I have now adjusted the cron job schedules so the metadata one happens first